File: src/ca/qc/bdeb/inf203/view/PositionnedSpriteContainer.java
package ca.qc.bdeb.inf203.view;

import ca.qc.bdeb.inf203.model.RepresentationImage;
import java.awt.Point;

/**
 * SpriteContainer avec une position
 * @author Nicolas Hurtubise, Guillaume Riou
 */
public class PositionnedSpriteContainer extends SpriteContainer {

    private Point position;

    public PositionnedSpriteContainer(Point position, RepresentationImage image, int animation) {
        super(image, animation);
        this.position = position;
    }

    public Point getCoordonnee() {
        return position;
    }
}

File: assets/js/view/positionned-sprite-container.js
/**
 * SpriteContainer avec une position
 * @author Nicolas Hurtubise, Guillaume Riou
 */
function PositionnedSpriteContainer(position, image, animation) {
    SpriteContainer.apply(this);
    /**
     * Point Position du spriteContainer.
     */
    this.position;

    this.image = image;
    this.animation = animation;
    this.position = position;
}

PositionnedSpriteContainer.prototype = new SpriteContainer();
